code/modules/uplink/uplink_items.dm

```diff
@@ -1762,6 +1762,12 @@ datum/uplink_item/stealthy_tools/taeclowndo_shoes
 	cost = 8
 	player_minimum = 20
 
+/datum/uplink_item/implants/omniChip
+	name = "OmniChip"
+	desc = "A Skillchip implant which will teach you all skills not related to martial arts. Comes with an autosurgeon."
+	item = /obj/item/autosurgeon/skillChip/omni
+	cost = 8 //Subject to Change before full PR Merge
```

Comment:
8 seems a bit much

Reply:
> 8 seems a bit much

Meant to be a placeholder value cause I have very little idea of how useful/not useful it is

Reply:
Reduced cost to 3